rosa_getting_started/rosa-quickstart-guide-ui.adoc

```diff
@@ -165,4 +165,4 @@ include::modules/rosa-getting-started-deleting-a-cluster.adoc[leveloffset=+1]
 
 * For information about setting up accounts and ROSA clusters without using AWS STS, see xref:../rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-getting-started-workflow.adoc#rosa-understanding-the-deployment-workflow[Understanding the ROSA deployment workflow].
 
-* For documentation on upgrading your cluster, see xref:../upgrading/rosa-upgrading.adoc#rosa-upgrading[Upgrading ROSA clusters].
+* For documentation on upgrading your cluster, see xref:../upgrading/rosa-upgrading-sts.adoc#rosa-upgrading-sts[Upgrading ROSA Classic clusters].
```

Comment:
```suggestion
* For more information on upgrading your cluster, see xref:../upgrading/rosa-upgrading-sts.adoc#rosa-upgrading-sts[Upgrading ROSA Classic clusters].
```
This is outside the scope of your changes, but per [the OCP docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-xref-attribute), the intro text for the xref should be changed as indicated.